cron.go

```diff
@@ -83,25 +85,26 @@ type FuncJob func()
 func (f FuncJob) Run() { f() }
 
 // AddFunc adds a func to the Cron to be run on the given schedule.
-func (c *Cron) AddFunc(spec string, cmd func()) error {
-	return c.AddJob(spec, FuncJob(cmd))
+func (c *Cron) AddFunc(spec string, cmd func(), n string) error {
+	return c.AddJob(spec, FuncJob(cmd), n)
 }
 
 // AddFunc adds a Job to the Cron to be run on the given schedule.
-func (c *Cron) AddJob(spec string, cmd Job) error {
+func (c *Cron) AddJob(spec string, cmd Job, n string) error {
```

Comment:
`n string` is not a very descriptive parameter. Someone seeing this library for the first time will likely not immediately understand what `n` represents. `name` would be better.

Also, the documentation currently says nothing about the name parameter, perhaps it should say "name is used for viewing purposes only".